templates/routes.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
```

Comment:
This is not going to work since the HttpRoute Kubernetes resource only allows up to 8 routes, and we have currently more than that. We should add another sidecar container following the Adapter Kubernetes Pattern.